crates/cheatcodes/tests/random.rs

```diff
@@ -0,0 +1,164 @@
+//! Contains various tests for `random*` and `fresh*` cheatcodes.
+
+use foundry_test_utils::{forgetest_init, str, util::OutputExt};
+
+// tests that `forge test` with a seed produces deterministic random values for uint and addresses.
+forgetest_init!(deterministic_randomness_with_seed, |prj, cmd| {
+    prj.wipe_contracts();
+    prj.add_test(
+        "DeterministicRandomnessTest.t.sol",
+        r#"
+import {Test, console} from "forge-std/Test.sol";
+
+contract DeterministicRandomnessTest is Test {
+
+    function testDeterministicRandomUint() public {
+        console.log(vm.randomUint());
+        console.log(vm.randomUint());
+        console.log(vm.randomUint());
+    }
+
+    function testDeterministicRandomUintRange() public {
+        uint256 min = 0;
+        uint256 max = 1000000000;
+        console.log(vm.randomUint(min, max));
+        console.log(vm.randomUint(min, max));
+        console.log(vm.randomUint(min, max));
+    }
+
+    function testDeterministicRandomAddress() public {
+        console.log(vm.randomAddress());
+        console.log(vm.randomAddress());
+        console.log(vm.randomAddress());
+    }
+}
```

Comment:
I guess I would prefer the name `freshUInt`, `freshAddress`, for example, rather than random. Maybe they can be included in the forge-std, but then just simplify down to teh `random` variants? The reason is because we want to use the same cheatcodes as Foundry directly, but for us they're not random, but symbolic. `fresh` captures both random and symbolic as sub-categories.

Reply:
the `randomAddress` and uint are already added, going to work on the `fresh*` cheatcodes and maybe look into aliasing existing